### ISSUE TITLE
[Hint, furture notice, feature request] Config Entry excavator_depletion_days kind of unprecise with getInt

### ISSUE THREAD

**Issue by mindforger** [BluSunrize/ImmersiveEngineering#536]
i did recognize the time for mineral depletion in days seems to not be calculated precisely :P

we nerved config wise the vein size to a smaller amount and decresed the speed to increase the RF cost per ore dramatically

but now the book only says a vein yield 0 days .. maybe because it only lasts half a day now XD
for the time beeing we decreased the speed further and in parallel decreased the power consumption but having a floating point value there to get a bit more precise calculations would be nice to have

**Comment by mindforger** [comment]
by a look at the calculation, do i read right? 76800 ores * 45 / 24000 = 144 Days ? how the ? can you probably explain the numbers for me? i am confused

**Comment by BluSunrize** [comment]
That value is only put in the config maps so the manual can access it. Also, this doesn't calculate in failure chances and stuff. So it's very rough and imprecise.
